frontend/src/components/common/SectionHeader.stories.tsx

```diff
@@ -14,45 +14,166 @@
  * limitations under the License.
  */
 
+import { Icon } from '@iconify/react';
+import { Button, Chip, IconButton, Typography } from '@mui/material';
 import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import { TestContext } from '../../test';
 import SectionHeader, { SectionHeaderProps } from './SectionHeader';
 
 export default {
-  title: 'SectionHeader',
+  title: 'common/SectionHeader', // Categorized under 'common'
```

Comment:
Thanks for fixing this.
I think these days you can just remove the title and it should figure it out?